blpapi.ts

```diff
@@ -0,0 +1,640 @@
+/// <reference path="./typings/tsd.d.ts" />
+
+'use strict';
+
+import assert = require('assert');
+import events = require('events');
+import util = require('util');
+
+import Promise = require('bluebird');
+import debug = require('debug');
+import _ = require('lodash');
+
+import path = require('path');
+/* tslint:disable:no-var-requires */
+// TODO REVIEW: Not sure how this module can be imported in a
+// 'TypeScript' fashion.
+var blpapijs = require(path.join(__dirname, '/build/Release/blpapijs'));
```

Comment:
@ericlu find a way to import the blpapijs module.

`process.env.NODE_PATH = path.join(__dirname, 'build/Release');`
`require('module').Module._initPaths();`
`import blpapijs = require('blpapijs');`

It is not ideal since it invokes the non-public function `_initPath` but it solves the type resolution issues that my initial code had from not being able to import the module.